src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -119,6 +119,17 @@ public void startTask(final String tableName, final TopicPartition topicPartitio
     }
   }
 
+  /**
+   * Create multiple ingesting tasks
+   *
+   * @param partitions collection of topic partitions
+   * @param topic2Table a mapping from topic to table
+   */
```

Comment:
you can drop the javadocs here since it is already present in interface

Reply:
Removed.